keras_hub/src/utils/transformers/export_gemma_to_safetensor.py

```diff
@@ -0,0 +1,134 @@
+import json
+import os
+
+import torch
+from safetensors.torch import save_file
+
+# Set the Keras backend to jax
+os.environ["KERAS_BACKEND"] = "jax"
```

Comment:
let's not do this, this is something we are going to export as part of the library. we actually need this to be able to run on all backends